src/api/eventTypes.js

```diff
@@ -173,3 +175,253 @@ export type RestartEvent = $ReadOnly<{|
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_allow_message_deleting'>,
+  allow_message_editing:
+    $PropertyType<InitialDataRealm, 'realm_allow_message_editing'>,
+  authentication_methods:
+    $PropertyType<InitialDataRealm, 'realm_authentication_methods'>,
+
+  // realm_available_video_chat_providers doesn't have an event because it
+  //   can only change when the server is restarted anyway; see
+  //   https://chat.zulip.org/#narrow/stream/378-api-design/topic/.60value.60.20in.20realm.20op.3A.20update/near/1276269
+
+  avatar_changes_disabled:
+    $PropertyType<InitialDataRealm, 'realm_avatar_changes_disabled'>,
+  bot_creation_policy:
+    $PropertyType<InitialDataRealm, 'realm_bot_creation_policy'>,
+  bot_domain:
+    $PropertyType<InitialDataRealm, 'realm_bot_domain'>,
+  community_topic_editing_limit_seconds:
+    $PropertyType<InitialDataRealm, 'realm_community_topic_editing_limit_seconds'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_create_private_stream_policy'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_create_public_stream_policy'>,
+  create_stream_policy:
+    $PropertyType<InitialDataRealm, 'realm_create_stream_policy'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_create_web_public_stream_policy'>,
+  default_code_block_language:
+    $PropertyType<InitialDataRealm, 'realm_default_code_block_language'>,
+
+  // realm_default_external_accounts doesn't have an event because it can
+  //   only change when the server is restarted anyway; see
+  //   https://chat.zulip.org/#narrow/stream/378-api-design/topic/.60value.60.20in.20realm.20op.3A.20update/near/1276269
+
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_default_language'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_default_twenty_four_hour_time'>,
+  // <insert_name>:
+  //   $PropertyType<InitialDataRealm, 'realm_delete_own_message_policy'>,
+  description:
+    $PropertyType<InitialDataRealm, 'realm_description'>,
+  digest_emails_enabled:
+    $PropertyType<InitialDataRealm, 'realm_digest_emails_enabled'>,
+  digest_weekday:
+    $PropertyType<InitialDataRealm, 'realm_digest_weekday'>,
+
+  // TODO: Number or boolean? Event doc and register-response doc disagree
+  disallow_disposable_email_addresses: empty,
+
+  edit_topic_policy:
+    $PropertyType<InitialDataRealm, 'realm_edit_topic_policy'>,
+  email_address_visibility:
+    $PropertyType<InitialDataRealm, 'realm_email_address_visibility'>,
+  email_auth_enabled:
+    $PropertyType<InitialDataRealm, 'realm_email_auth_enabled'>,
+  email_changes_disabled:
+    $PropertyType<InitialDataRealm, 'realm_email_changes_disabled'>,
+  emails_restricted_to_domains:
+    $PropertyType<InitialDataRealm, 'realm_emails_restricted_to_domains'>,
+  realm_giphy_rating:
+    $PropertyType<InitialDataRealm, 'realm_giphy_rating'>,
```

Comment:
Hmm, I wonder if this in the docs is accurate, or if it's actually `giphy_rating`. Seems like the general naming scheme in effect here would want it to be the latter.